isis/src/lro/objs/LroWideAngleCamera/unitTest.cpp

```diff
@@ -33,7 +33,7 @@ using namespace Isis;
 
 // 2013-12-19 We will have new CK kernels with temperature dependence coming
 // that will affect the lat/lon values here.  Also must get a new FK, IK and
-// IAK.  Don;t forget to move the new test cube to $ISISROOT/lro/testData!!!!
+// IAK.  Don't forget to move the new test cube to $ISISROOT/lro/unitTestData!!!!
```

Comment:
This doesn't look like the right path. Should it be $ISISTESTDATA/isis/src/lro/unitTestData

Reply:
Good catch! I'll fix this.